functions/utils/index.js

```diff
@@ -13,4 +13,71 @@ const getUrl = async srcFile => {
   }
 };
 
+const secondsToDhms = (seconds) => {
+  seconds = Number(seconds);
+  var d = Math.floor(seconds / (3600 * 24));
```

Comment:
Any reason these are `var` not `let`? Maybe just copy/paste?

Reply:
😂 yup